e2e/conf/protractor.base.conf.js

```diff
@@ -18,7 +18,10 @@ exports.config = {
     testPath +  '/**/*.feature'
   ],
   capabilities: {
-    'browserName': 'chrome'
+    'browserName': 'chrome',
+    'chromeOptions': {
+      'args': ['--no-sandbox']
+    }
```

Comment:
I agree with @jludvice , probably best to move to `e2e/conf/protractor.ipaas-qe.conf.js` eventually, but for now is not a big deal at all. 👍